### ISSUE TITLE
Security vulnerability - nth-check dependency

### ISSUE THREAD

**Issue by gablittledevshop** [visualfanatic/vue-svg-loader#175]
Running `npm audit` in my project reveals a vulnerability with the `nth-check` dependency  https://github.com/advisories/GHSA-rp65-9cf3-cjxr . This seems to be resolved already in other dependencies, `css-stream` https://github.com/fb55/css-select/issues/607, but not with svg-to-vue.

**Comment by frankykubo** [comment]
Yeah, it would be very nice if maintainer resolved this. However it looks like that this project is not maintained anymore.
